src/test/java/api/loans/agetolost/ScheduledAgeToLostFeeChargingApiTest.java

```diff
@@ -319,6 +324,35 @@ public void shouldCloseLoanWhenNoFeesToCharge() {
   }
 
   @Test
+  public void shouldNotChargeOverdueOnCheckinWhenAgedToLostAndRefundFeePeriodPassed() {
+    // for a loan that charges both lost fees and overdue fines
+    // where the policies are set such that:
+    // 1. the item charges overdue fees on checkin
+    // 2. A refund expiration date is set
+    // 3. The item is aged to lost
+    // 4.  the item is checked in after the refund expiration date has passed
+    // THEN the loan SHOULD NOT have overdue charges
+    IndividualResource overDueFinePolicy = overdueFinePoliciesFixture.facultyStandard();
+
+    IndividualResource lostItemPolicy = lostItemFeePoliciesFixture.ageToLostAfterOneWeek();
+
+    val result = ageToLostFixture.createLoanAgeToLostAndChargeFeesWithOverdues(lostItemPolicy, overDueFinePolicy);
+
+    assertThat(result.getLoan().getJson(), isLostItemHasBeenBilled());
+
+    UUID loanId = result.getLoan().getId();
+
+    // the creation function ages the loan eight weeks into the future.  
+    // it must be checked in after that timeframe to properly examine the 
+    // overdue charges
+    final DateTime checkInDate = now(UTC).plusWeeks(9);
+    mockClockManagerToReturnFixedDateTime(checkInDate);
+    checkInFixture.checkInByBarcode(result.getItem(), checkInDate);
+
+    assertThat(loansFixture.getLoanById(loanId), hasNoOverdueFine());
+
```

Comment:
Personally, I would remove this blank line as I don't think it improves readability